nose_unit.cfg

```diff
@@ -6,5 +6,5 @@ nologcapture=1
 verbosity=2
 where=yt
 with-timer=1
-ignore-files=test_load_errors.py
+ignore-files=(test_load_errors.py|test_load_sample.py)  # tests written for pytest
```

Comment:
If you ran it with -vv to test it locally, you'd see that your comment is treated as a part of the regex:

```
$ nosetests -vv -c nose_unit.cfg --collect-only tests/
...
nose.config: INFO: Ignoring files matching ['(test_load_errors.py|test_load_sample.py)  # tests written for pytest']
...
```

Drop it and it will work.

Reply:
sorry I never had the patience to learn nose... Thanks a lot !